migration.md

```diff
@@ -1,5 +1,24 @@
 # Migration
 
+## 0.6.0
+
+Last minor we introduced a changed saying "Fixed dependencies on @lion".
+We fixed our dependencies on lion inside simba, because lion does so internally as well.
+This way, we ensure users don't get duplicate installs when Lion releases a new version but internally another Lion packages is still depending on the older version.
+
+However, the Lion team took another look at how this was working for its users and came to the conclusion that this approach is flawed.
+We're better served if the versions are using carets (^), this will prevent lots of cases where duplicate installations are inevitable, and also allows users to import from Lion directly when using Simba
+For more information, [refer to Versioning docs](https://lion-web.netlify.app/docs/rationales/versioning/) in Lion.
+
+What does this mean for you?
+
+- Please do not use fixed npm versions of simba, just use `^`.
+- This will allow importing directly from latest Lion without issues, again, using `^`
+- Less likely to get these hard to troubleshoot bugs due to undeduped duplicate installations. `^`s, assuming semver is followed properly, are way more developer friendly, as it turns out.
+
+We decided to release a minor for simba for this, just in case this dependency resolution is breaking somehow, it's a bit unpredictable with this change whether it would break something for someone.
```

Comment:
not to need to confuse people with minor/major, I'd call a breaking version bump or so

Reply:
Changed